src/domains/marketing/UsageData.js

```diff
@@ -24,15 +24,19 @@ export function UsageData() {
                     Users
                   </dt>
                   <dd className="order-1 text-5xl font-bold tracking-tight text-green-600">
-                    {data.usageData.userCount}
+                    {new Intl.NumberFormat("en-IN").format(
+                      data.usageData.userCount
+                    )}
                   </dd>
                 </div>
                 <div className="flex flex-col border-t border-stone-100 p-6 text-center sm:border-0 sm:border-l">
                   <dt className="order-2 mt-2 text-lg font-medium leading-6 text-stone-500">
                     Unfollows
                   </dt>
                   <dd className="order-1 text-5xl font-bold tracking-tight text-green-600">
-                    {data.usageData.unfollowedCount}
+                    {new Intl.NumberFormat("en-IN").format(
+                      data.usageData.unfollowedCount
```

Comment:
Have you tested by hardcoding a high number such as `25666` here? I imagine you might get a problem with hydration 🤔